occupi-backend/pkg/router/router.go

```diff
@@ -57,6 +57,8 @@ func OccupiRouter(router *gin.Engine, db *mongo.Client) {
 		auth.POST("/register", middleware.UnProtectedRoute, func(ctx *gin.Context) { handlers.Register(ctx, appsession) })
 		auth.POST("/verify-otp", middleware.UnProtectedRoute, func(ctx *gin.Context) { handlers.VerifyOTP(ctx, appsession) })
 		auth.POST("/logout", middleware.ProtectedRoute, func(ctx *gin.Context) { handlers.Logout(ctx) })
-		// auth.POST("/reset-password", middleware.UnProtectedRoute, func(ctx *gin.Context) { handlers.ForgotPassword(ctx) })
+		// it's typically used by users who can't log in because they've forgotten their password.
+		auth.POST("/forgot-password", func(ctx *gin.Context) { handlers.ResetPassword(ctx, appsession) })
```

Comment:
please add unprotected route middleware for this endpoint, as we would not want users who are logged on accessing this endpoint

Reply:
okay i will add it but there's really no need to because either way, whether you're logged in or not and you still forgot your password, the system shouldnt necessarily "lock" you out

Reply:
> okay i will add it but there's really no need to because either way, whether you're logged in or not and you still forgot your password, the system shouldn't necessarily "lock" you out

yes true, we'll discuss this week